doc/Developers.md

```diff
@@ -201,6 +201,49 @@ The scripts for publishing Princess and SMTInterpol are available
 at the root of the [Ivy Repository](https://svn.sosy-lab.org/software/ivy).
 
 
+### Publishing Yices2
+
+Yices2 consists of two components: the solver binary and the Java components in JavaSMT.
+The Java components were splitt from the rest of JavaSMT because of the GPL.
+
+#### Publishing the solver binary for Yices2
+
+Prepare gperf and gmp (required for our own static binary):
+```
+wget http://ftp.gnu.org/pub/gnu/gperf/gperf-3.1.tar.gz && tar -zxvf gperf-3.1.tar.gz && cd gperf-3.1 && ./configure && make
```

Comment:
Is this simplyfied version of how to build (compared to the original build information in compile.sh) working flawlessly?

Reply:
I will test it again. Maybe I have overlooked something.

Reply:
I was just asking because in the compile.sh of Yices2 provided by the author has some extra flags. I didn't test (but can if you want me to).

Reply:
The script compile.sh was written by a BA student and possibly copied together from other scripts.
I know that `-fpic` is important, but I could not yet crash the application.
Performance could also be different on larger queries.
I will check again.

Reply:
I tested the concurrency with those flags. Maybe without them its working? I'll test tomorrow. Want me to run it through CPAchecker to check the performance?